DummyProject/DummyProject/AppDelegate.swift

```diff
@@ -16,7 +17,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                      didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         let configuration = SDKConfiguration(
             appCode: "LS",
-            clientSecret: "$2y$10$O7HK3Afr1PZH3WTiQ7bTg.kfcle88e/n9GqrcCp7qWH8Rvv.Ojl/C")
```

Comment:
We should find a way to scrub this from the repo as I don't think we should be committing secrets.

Reply:
Yeah, totally agree with you. I think we can approach it with below options at the moment.
Option 1. We can leave fake appCode, clientSecret and apiUrl here and let QA fills these up by themselves in codes.
Option 2. Create a plist file which contains appCode, clientSecret and apiUrl.

The option 2 is a bit redundant as the developers still can access the plist file via the Xcode project, so I'd prefer option 1, but we have to make sure QA knows how to change these configurations in codes.

Reply:
Yeah, Option 1 sounds better, we can write a guide for them on confluence

Reply:
Thanks. I removed the configuration details from the repo and attach the tutorial to the ticket.